doc/classes/ClassDB.xml

```diff
@@ -16,6 +16,14 @@
 				Returns [code]true[/code] if objects can be instantiated from the specified [param class], otherwise returns [code]false[/code].
 			</description>
 		</method>
+		<method name="class_call_static_method" qualifiers="vararg">
+			<return type="Variant" />
+			<param index="0" name="class" type="StringName" />
+			<param index="1" name="method" type="StringName" />
+			<description>
+				Calls a static method on a class. Can be made into a callable by binding a class and a method.
```

Comment:
I don't understand the 2nd sentence - is this perhaps a left over reference to the other PR that uses callables?

I think we could just drop that:

```suggestion
				Calls a static method on a class.
```

Reply:
Oh yeah, it's a left over reference from ClassDB::class_get_static_method PR. Thanks